h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -450,7 +452,35 @@ public static Frame parse_test_file( Key outputKey, String fname, ParseSetupTran
     return ParseDataset.parse(outputKey, new Key[]{nfs._key}, true, guessedSetup);
   }
 
-  protected Frame parse_test_file( String fname, String na_string, int check_header, byte[] column_types) {
+  private static final String TEST_DATA_PATH_PREFIX_KEY = "H2O_TEST_DATA_PATH_PREFIX";
```

Comment:
since the constant is used only once I think its not necessary
if you want to keep it should be named SMALLDATA_PATH_PREFIX because thats what it does

Reply:
It's not just smalldata. If you sync bigdata or bigdata-laptop, it will work as well. 

But I agree and will remove the constant. Totally not worth it.